managed/CustomGroup__event_messages_settings.mgd.php

```diff
@@ -0,0 +1,256 @@
+<?php
+/*
+ * Copyright (C) 2025 SYSTOPIA GmbH
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as published by
+ *  the Free Software Foundation in version 3.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+declare(strict_types = 1);
+
+use CRM_Eventmessages_ExtensionUtil as E;
+use Civi\Api4\SiteEmailAddress;
+
+return [
+  [
+    'name' => 'CustomGroup_event_messages_settings',
+    'entity' => 'CustomGroup',
+    'cleanup' => 'always',
+    'update' => 'unmodified',
+    'params' => [
+      'version' => 4,
+      'values' => [
```

Comment:
Attribute `table_name` is missing. This should/must be there for two reasons:

1. Equal db schema on all installations.
2. The table name `civicrm_value_event_messages_settings` is (currently) used at some places.

Number 2 is the reason for the test failure.